purchase_allowed_product/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,8 @@
+* Chafique Delli <chafique.delli@akretion.com>
+* Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+* Hieu, Vo Minh Bao <hieu.vmb@komit-consulting.com>
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * Manuel Calero
+  * Pedro M. Baeza
+  * Darius Žižys <darius@vialaurea.lt>
```

Comment:
You should add a line break before your name and the untab to get out of Tecnativa's company contributors

Reply:
Sorry. I'll fix that... How hard is to get a job at Tecnativa? :smile: